lib/site_prism/section.rb

```diff
@@ -62,16 +62,20 @@ def native
 
     private
 
-    def find_first(*find_args)
-      root_element.find(*find_args)
+    def _find(*find_args)
+      page.find(*find_args)
+    end
+
+    def _all(*find_args)
+      page.all(*find_args)
     end
 
     def element_exists?(*find_args)
-      root_element.has_selector?(*find_args) unless root_element.nil?
+      page && page.has_selector?(*find_args)
```

Comment:
Is it possible to have a section without root element?

Reply:
These were just direct replacements. But no you cannot have a section without a root_element. I'll alter this.